jwst/model_blender/rules.py

```diff
@@ -0,0 +1,65 @@
+from datetime import time
+
+import numpy as np
+
+
+__all__ = ['RULE_FUNCTIONS', 'Blender', 'make_blender']
+
+
+def multi(vals):
+    """
+    This will either return the common value from a list of identical values
+    or 'MULTIPLE'
+    """
+    uniq_vals = list(set(vals))
+    num_vals = len(uniq_vals)
+    if num_vals == 0:
+        return None
+    if num_vals == 1:
+        return uniq_vals[0]
+    if num_vals > 1:
+        return "MULTIPLE"
+
+
+def _isotime(time_str):
```

Comment:
This function doesn't appear to be used - can we remove it?

Reply:
Thanks for the catch. I removed this in d08c42dc5 It was previously used by the `bad_min/maxtime` and could have been removed when those were removed.